### PR TITLE
feat(l10n): add Danish localization

### PR DESCRIPTION
**Pull request by ddfreiling**
Note: In the English translation I assumed in `packages/ace-report-axe/src/l10n/locales/da.json` that it was a mistake having the key "area-alt" translated to "Image maps", is this correct?

I will have Marianne, who reviewed my Danish translation of axe-core (https://github.com/dequelabs/axe-core/pull/1876), review this as well. I will let you know when it is done.

### CODE REVIEW

File: packages/ace-report-axe/src/l10n/locales/da.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        "aria-required-parent": "ARIA",
+        "aria-roles": "ARIA",
+        "aria-valid-attr": "ARIA",
+        "aria-valid-attr-value": "ARIA",
+        "button-name": "Formularer",
+        "checkboxgroup": "Formularer",
+        "color-contrast": "Farve",
+        "definition-list": "Lister",
+        "dlitem": "Lister",
+        "document-title": "Sidens Titel",
```
Comment (MyDK) [inline packages/ace-report-axe/src/l10n/locales/da.json]: "document-title": "Sidens titel",

File: packages/ace-report-axe/src/l10n/locales/da.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        "duplicate-id": "Identifikator",
+        "empty-heading": "Overskrifter",
+        "epub-type-has-matching-role": "ARIA roller",
+        "frame-title": "Indsatte Sider",
+        "frame-title-unique": "Indsatte Sider",
```
Comment (MyDK) [inline packages/ace-report-axe/src/l10n/locales/da.json]: "frame-title-unique": "Indsatte sider",

File: packages/ace-report-axe/src/l10n/locales/da.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        "button-name": "Formularer",
+        "checkboxgroup": "Formularer",
+        "color-contrast": "Farve",
+        "definition-list": "Lister",
+        "dlitem": "Lister",
+        "document-title": "Sidens Titel",
+        "duplicate-id": "Identifikator",
+        "empty-heading": "Overskrifter",
+        "epub-type-has-matching-role": "ARIA roller",
+        "frame-title": "Indsatte Sider",
```
Comment (MyDK) [inline packages/ace-report-axe/src/l10n/locales/da.json]: "frame-title": "Indsatte sider",

File: packages/ace-core/src/l10n/locales/da.json [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            "testdesc": "Sikrer at '{{name}}' metadata-elementet er til stede"
+        },
+        "pagesourceviolation": {
+            "kbtitle": "Sidenavigation",
+            "resdesc": "Tilføj et 'dc:source' metadata-element til pakke-dokumentet",
+            "ruledesc": "Publikationer med sideskift skal angive et 'dc:source' metadata-element",
+            "testdesc": "Sikrer at kilden til sideskift er identificeret"
+        },
+        "titleviolation": {
+            "kbtitle": "EPUB Titel",
```
Comment (MyDK) [inline packages/ace-core/src/l10n/locales/da.json]: "kbtitle": "EPUB titel",

File: packages/ace-report/src/l10n/locales/da.json [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    "pagenav": "Sidenavigation",
+    "pubmetadatacaption": "Publikationens metadata.",
+    "report-desc": "Rapport over automatiseret tjek af tilgængelighed i EPUB'en",
+    "report-title": "Ace Rapport",
+    "resetfilters": "Nulstil filtre",
+    "role": "Rolle",
+    "rule": "Regel",
+    "ruleset": "Regelsæt",
+    "serious": "Seriøs",
+    "snippet": "Stump:",
```
[the remark this answers is not in the document]
Comment (ddfreiling) [reply packages/ace-report/src/l10n/locales/da.json]: I have kept colon, where it occurred at the end of the text in the English translation

**Comment by ddfreiling** [comment]
Updated after review, should be ready for merge now. Thanks @MyDK !

**Comment by danielweck** [comment]
Hello Daniel, thank you for this contribution!
Would you be interested in translating Ace App (the desktop user interface for Ace)?
If so, there are instructions in this wiki page:
https://github.com/daisy/ace-gui/wiki/Localization
Regards, Daniel

**Comment by danielweck** [comment]
Merging this PR into a staging branch `feat/l10n-da` in preparation for final merging into `master` and `ace-next` (checks pending)
Thank you! I will let you know how it goes.

**Comment by danielweck** [comment]
Now merged into `master`, thank you very much!
https://github.com/daisy/ace/pull/316

**Comment by ddfreiling** [comment]
No problem! I will look into translating the Ace App, when I find the time.
- Best regards, Nota in Denmark

**Comment by danielweck** [comment]
Hello, I have now also merged your Danish translation into the "Ace next" branch ( https://github.com/daisy/ace/pull/314 ), which introduces 2 new labels. So, could you please take a look and let me know what strings to use, and I will fix the branch:

https://github.com/daisy/ace/pull/314/commits/72c8ded9f3c972039b13ef83795ff77d5a61e7f7#diff-5530ffd4b88ca1f4ad6ea40abbc0f976

Thank you!!

**Comment by ddfreiling** [comment]
```
"audio-caption": "Lyd",
...
"style": "Styling",
```
Do you want a PR for this?

**Comment by danielweck** [comment]
No thank you, I am integrating changes now :)

**Comment by danielweck** [comment]
Note that I added credits in the wiki page with your contributor GitHub handle:
https://github.com/daisy/ace/wiki/Localization

**Pull request merged**